stage/stage.go

```diff
@@ -44,6 +44,8 @@ type Stage struct {
 	PostStageShellScripts []string `json:"post_stage_scripts,omitempty"`
 	// Run shell scripts after executing each query.
 	PostQueryShellScripts []string `json:"post_query_scripts,omitempty"`
+	// Run shell scripts after finishing full query cycle runs each query.
+	PostQueryCycleShellScripts []string `json:"post_query_cycle_scripts,omitempty"`
```

Comment:
@steveburnett for doc

Reply:
A full cycle here means when we set `cold_runs` and `warm_runs`, each query in the benchmark will be run `cold_runs + warm_runs` times in total.

`post_query_scripts` will be called after each query execution, `post_query_cycle_scripts` will be called after all the `cold_runs` and `warm_runs` are done for a unique query.